Dockerfile

```diff
@@ -6,11 +6,11 @@ RUN dnf update -y && \
     libXi-devel libXrandr-devel libXcursor-devel libXdamage-devel libXinerama-devel -y && \
     dnf clean all -y
 
-RUN git clone https://gitlab.gnome.org/gnome/gtk.git --depth=1 && \
+RUN git clone https://gitlab.gnome.org/gnome/gtk.git --depth=1 --branch gtk-4-12 && \
```

Comment:
Please no. We offer GTK 4.14 bindings already

Reply:
Okay, it didn't work anyway. Will revert this.

Reply:
I hate libadwaita depending on appstream but you can install it already to avoid having to build that in the image

Reply:
You mean with dnf?

Reply:
yes